### PR TITLE
replaced two backticks with one backtick

### PR DESCRIPTION
**Pull request by OskarStark**
Fixes #3794

### CODE REVIEW

**Comment by core23** [comment]
Should we fix the other markdown files too?

**Comment by greg0ire** [comment]
Offending files can be found like this : `ag '[^`]``\w' --ignore vendor --markdown`

```
UPGRADE-2.1.md
10:        widget (if the ``multiple`` option is set)
12:      * sonata_type_model_list : this type replaces the option ``edit = list`` provided as
13:        a 4th argument on the ``sonata_type_model``

CHANGELOG.md
17:- Add missing Route constructor parameters to ``RouteCollection:add`` method
18:- Add the ``hasRoute`` method to the AdminInterface
19:- Integration of KNPMenu for the admin menu. This integration is reset when the standard layout
20:``standard_layout.html.twig`` is overriden. The KNPMenu is available in ``sonata_menu.html.twig`` template.
21:- Add ``getFieldOption``, ``setFieldOption`` methods to the FilterInterface
22:- Add the ``getFilterFieldDescription`` method to the AdminInterface
23:- Add the ``getMaxPageLinks``, ``setMaxPageLinks`` methods to the PagerInterface
27:- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
28:- CSS class ``sonata-autocomplete-dropdown-item`` is not automatically added to dropdown
29:autocomplete item in ``sonata_type_model_autocomplete``, use option ``dropdown_item_css_class``
34:- Remove ``btn-outline``` from doctrine-orm-admin form actions buttons

UPGRADE-3.0.md
17:CSS class ``sonata-autocomplete-dropdown-item`` is not automatically added to dropdown autocomplete item in ``sonata_type_model_autocomplete``, use option ``dropdown_item_css_class`` to set the CSS class of dropdown item.
19:## Standard Layout
20:``sonata_wrapper`` block was moved and is now inside the ``.wrapper`` div of admin lte theme.
24:The inline validation has been migrating to CoreBundle. Just rename ``Sonata\AdminBundle\Validator\ErrorElement`` to ``Sonata\CoreBundle\Validator\ErrorElement``
40:The ``ModelsToArrayTransformer`` has been renamed to ``LegacyModelsToArrayTransformer``. ``ModelsToArrayTransformer`` should be only be used with SF2.7+

UPGRADE-2.3.md
10: - ``standard_layout.html.twig`` has been updated (refactored layout accordingly to the theme). Some blocks have been moved, and the sonata_side_nav twig block became the main menu.
15: - ``configureSideMenu`` and ``buildSideMenu`` methods of the ``Admin`` class have been deprecated; they are replaced by ``configureTabMenu`` and ``buildTabMenu``.
19: - ``configureShowField`` has been deprecated since a while now, use ``configureShowFields`` instead => add a ``s``.
```

Can you fix all of them?

**Comment by OskarStark** [comment]
sure will do

**Comment by OskarStark** [comment]
btw could we add this to our travis job?

**Comment by Soullivaneuh** [comment]
> btw could we add this to our travis job?

What do you mean? The @greg0ire check?

Why not. It could be added under the docs target of the Makefile.

Feel free to provide a PR on dev-kit! :+1:

**Comment by OskarStark** [comment]
> What do you mean? The @greg0ire check?
yes

**Comment by greg0ire** [comment]
Maybe there is a markdown linter that would catch that kind of error…

**Comment by OskarStark** [comment]
@greg0ire which command did you execute? `ag` ?

**Comment by greg0ire** [comment]
> @greg0ire which command did you execute? ag ?

Yes, it's grep on steroids : https://github.com/ggreer/the_silver_searcher/blob/master/doc/ag.1.md

**Comment by Soullivaneuh** [comment]
@OskarStark This kind of PR should not be milestoned. It's just documentation update.

**Comment by greg0ire** [comment]
So "patch" makes no sense either, does it?

File: UPGRADE-2.1.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ UPGRADE FROM 2.0 to 2.1
...
       * sonata_type_model : this type now only renders a standard select widget or a list
-        widget (if the ``multiple`` option is set)
+        widget (if the `multiple` option is set)
 
-      * sonata_type_model_list : this type replaces the option ``edit = list`` provided as
-        a 4th argument on the ``sonata_type_model``
+      * sonata_type_model_list : this type replaces the option `edit = list` provided as
+        a 4th argument on the `sonata_type_model`
 
+`
```
Comment (OskarStark) [inline UPGRADE-2.1.md]: ups

**Comment by Soullivaneuh** [comment]
Thank you @OskarStark

**Pull request merged**